test/RPCSession.h

```diff
@@ -47,7 +46,8 @@ class IPCSocket : public boost::noncopyable
 
 private:
 	std::string m_path;
-	HANDLE  m_socket;
+	HANDLE m_socket;
+	TCHAR m_readBuf[512000];
```

Comment:
Moving this from the method, from being a stack variable to be a on the heap. Also reduced from 5Mb to 512k.